App/Redux/TextileRedux.js

```diff
@@ -54,11 +59,22 @@ export const handleImageAdded = (state, {image, remotePayloadPath}) => {
   return state.merge({ images: { items } })
 }
 
+export const handleImageSubmittedForUpload = (state, {uploadId}) => {
+  const existingItems = state.images.items ? state.images.items : []
+  const items = existingItems.map(item => {
+    if (item.uploadId === uploadId) {
```

Comment:
If I understand this, this reducer is meant to update the image data for the image that has just been submitted for upload, updating it's state with the `uploadId`. So a couple issues... 

The image should be one of the action arguments (you're only extracting `uploadId` above. You'll have to find the image to update using `remotePayloadPath`, the same way we do below in `handleImageProgress`. Once you find it, you want to add a new property to it, the `uploadId`. I don't see that happening anywhere.